js/piwik.js

```diff
@@ -3223,7 +3225,14 @@ if (typeof window.Piwik !== 'object') {
                     (msToExpire ? ';expires=' + expiryDate.toGMTString() : '') +
                     ';path=' + (path || '/') +
                     (domain ? ';domain=' + domain : '') +
-                    (secure ? ';secure' : '');
+                    (isSecureCookie ? ';secure' : '');
+            }
+
+            /*
+             * Set cookie secure flag
+             */
+            function setSecureCookie(value) {
+                isSecureCookie = value == 1 ? true : false;
```

Comment:
@sgiehl Hi there! I did the changes you requested, including the minifying. I can see there on my piwik branch but they are not being reflected in the PR. Could you tell me why?